json-rpc/src/tests/unit_tests.rs

```diff
@@ -1318,3 +1326,11 @@ fn execute_batch_and_get_first_response(
         .remove(0)
         .unwrap()
 }
+
+fn gen_string(len: usize) -> String {
```

Comment:
https://docs.rs/rand/0.7.3/rand/trait.Rng.html#method.fill maybe faster? I don't actually know

Reply:
not sure, but prefered Alphanumeric string, all random may produce double quote, which may break json format. so will just keep this version